CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - Fix Kaoto Datamapper editor on Windows (issue when attaching an xsd)
 - Upgrade to Kaoto 2.4.0-RC1
+- Add the possibility to enable experimental features (drag & drop)
```

Comment:
this PR is also adding a setting for nodeToolbarTrigger

Reply:
and given that commit message and title of PR is also mentioning only experimental feature, is it that you do not wanted to have the setting for nodeToolbarTrigger? In this case, instead of adding something in changelog (and updating commit message), need to remove in package.json